docs/en/interfaces/formats.md

```diff
@@ -1109,9 +1147,13 @@ struct Message {
 }
 ```
 
-Deserialization is effective and usually does not increase the system load.
+To insert data into [Nested](../sql-reference/data-types/nested-data-structures/nested.md) columns as an array of structs values you must enable the [input_format_parquet_import_nested](../operations/settings/settings.md#input_format_parquet_import_nested) setting.
 
-See also [Format Schema](#formatschema).
+You can select data from a ClickHouse table and save them into some file in the CapnProto format by the following command:
+
+``` bash
+$ clickhouse-client --query = "SELECT * FROM capnp_tuples FORMAT CapnProto SETTINGS format_schema = '$CLIENT_SCHEMADIR/02030_capnp_tuples:Message'"
```

Comment:
```suggestion
$ clickhouse-client --query = "SELECT * FROM test.hits FORMAT CapnProto SETTINGS format_schema = 'schema:Message'"
```
Let's use an example with `test.hits` and corresponding schema from previous example.